examples/rsvp-6-vevt/index.mjs

```diff
@@ -14,6 +15,9 @@ const makeRSVP = async ({ hostLabel, name, reservation, timeLimit }) => {
   const sbal = stdlib.parseCurrency(100);
   const accHost = await stdlib.newTestAccount(sbal);
   accHost.setDebugLabel(hostLabel);
+  if (stdlib.connector != 'ALGO') {
```

Comment:
If I recall, there's no danger to setting it anyways, it just gets ignored

Reply:
I use the pattern to avoid a warning when connected to ALGO. But if cleaner codebase is more desirable, I can remove the condition.